src/main.js

```diff
@@ -124,13 +150,15 @@ let load = base.loader({
   },
 
   api: {
-    requires: ['cfg', 'WorkerType', 'Secret', 'ec2', 'stateContainer', 'validator', 'publisher', 'influx'],
-    setup: async ({cfg, WorkerType, Secret, ec2, stateContainer, validator, publisher, influx}) => {
+    requires: ['cfg', 'WorkerType', 'WorkerState', 'amiSet', 'Secret', 'ec2', 'validator', 'publisher', 'influx'],
+    setup: async ({cfg, WorkerType, WorkerState, amiSet, Secret, ec2, validator, publisher, influx}) => {
+
```

Comment:
As above, I don't think you want WorkerState in here (@jhford please correct me if I'm wrong!)